### ISSUE TITLE
Make prettier & work with Light/Dark Mode switches

### ISSUE THREAD

**Issue by tryonlinux** [tryonlinux/kanban-board-sn#2]
Need to make it prettier as well as work with dark and light mode switches.

**Comment by tryonlinux** [comment]
Works on the default theme and most of the other themes. Need to figure out how to naturally work with all themes

**Comment by tryonlinux** [comment]
also make the item boxes themselves respect theme

**Comment by Donnnno** [comment]
It would be great to have a more broad support for themes, if it's possible to do!
Anyways, thanks for the simple kanban board :)

**Comment by tryonlinux** [comment]
Yep that's the plan for when I fix this issue it should work with most extensions. Just gotta find the time to do it haha. 

And you're welcome it was a pleasure to create!

**Comment by Donnnno** [comment]
Haha, take your time! Thanks :^)
